maple-sdk/introduction.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+
+`maple-js` is the JavaScript SDK for Maple Protocol, a comprehensive and user-friendly toolkit designed to facilitate seamless interaction with Maple Protocol's blockchain-based smart contracts. This SDK provides developers with the necessary tools to integrate with the Maple Protocol ecosystem and it simplifies the process of connecting to the blockchain, managing accounts, submitting transactions, and handling events generated by smart contracts.
+
+By utilizing the JavaScript SDK for Maple Protocol, developers can efficiently create robust and secure dApps that leverage the full potential of Maple Protocol's smart contracts, fostering innovation and driving the adoption of decentralized finance solutions.
+
+## Features
+
+1. **Collateral management** through `collateralLockerImports`
+2. **Debt management** with versioned support (v1, v2, and v3) through `debtLockerV1Imports`, `debtLockerV2Imports`, and `debtLockerV3Imports`
+3. **Funding management** using `fundingLockerImports`
+4. **Late fee calculation** with `lateFeeCalcImports`
+5. **Liquidity management** using `liquidityLockerImports`
+6. **Loan management** with versioned support (v1, v2, v3, v301, v4, and v5) through `loanV1Imports`, `loanV2Imports`, `loanV3Imports`, `loanV301Imports`, `loanV4Imports`, and `loanV5Imports`
+7. **Global settings and configurations management** in versioned support (v1, v2, and v2v2) through `mapleGlobalsImports`, `mapleGlobalsV2Imports`, and `mapleGlobalsV2V2Imports`
+8. **Maple Protocol rewards management** with `mapleRewardsImports`
+9. **Maple Protocol token management** with `mapleTokenImports`
+10. **XMPL token management** with `xmplImports`
+11. **Pool management** with versioned support (v1 and v2) through `poolV1Imports` and `poolV2Imports`
+12. **Premium calculation** using `premiumCalcImports`
+13. **Repayment calculation** with `repaymentCalcImports`
+14. **Stake management** using `stakeLockerImports`
+15. **Pool management** using `poolImports`
+16. **Withdrawal management** with `withdrawalManagerImports`
+17. **Environment mocks** for testing through `environmentMocksImports`
```

Comment:
Does webapp currently use `EnvironmentMocks` ? Only thing is we do have the `ERC20.sol` there together with `BPool` stuff
https://github.com/maple-labs/environment-mocks/tree/main/contracts

Not sure if we should include this